### PR TITLE
allowing hook file to live in project root directory

### PR DESCRIPTION
**Pull request by jdoiro3**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+mergedeep==1.3.4
+mkdocs==1.2.3
+packaging==21.3
+pyparsing==3.0.6
+python-dateutil==2.8.2
+PyYAML==6.0
+pyyaml-env-tag==0.1
+six==1.16.0
+watchdog==2.1.6
+zipp==3.6.0
```
Comment (aklajnert) [inline requirements.txt]: This file is not needed.

**Review by aklajnert — COMMENTED** [review]
Thanks for the PR, can you also look at the tests, why they are failing?

**Comment by jdoiro3** [comment]
Will do. The change works locally but will investigate what's going on with the tests. 

I also checked out a previous commit, running pytest, and they failed, which is a bit confusing.

**Review by jdoiro3 — COMMENTED** [review]
Leaving out requirements

**Comment by aklajnert** [comment]
OK, it turns out that the tests were broken due to dependency change. I've fixed them - please update your branch and it should work.

**Comment by jdoiro3** [comment]
Looks like the tests are passing with both updates. Let me know if you'd like me to do anything else. I was thinking about updating the README but not sure if you'd rather do it.

File: mkdocs_simple_hooks/__init__.py [hunk 2]
```diff
@@ -39,9 +40,12 @@ def handle_hook(self, *args, hook, **kwargs):
         return hook(*args, **kwargs)
 
     def _get_function(self, hook_path, warns):
-        cwd = os.getcwd()
-        if cwd not in sys.path:  # pragma: no cover
-            sys.path.append(cwd)
+        cwd = Path(os.getcwd())
+        root_dir = cwd.parent
+        if str(cwd) not in sys.path or str(root_dir) not in sys.path:  # pragma: no cover
```
Comment (aklajnert) [inline mkdocs_simple_hooks/__init__.py]: These should be two separate conditions. If `cwd` not in path - append only `cwd`. If `root_dir` not in path - append only `root_dir`.

**Comment by aklajnert** [comment]
> Looks like the tests are passing with both updates. Let me know if you'd like me to do anything else. I was thinking about updating the README but not sure if you'd rather do it.

Yeah, I think README could mention that the hooks can be defined in the project root.

File: mkdocs_simple_hooks/__init__.py (hunk 2 above)
Comment (jdoiro3) [reply mkdocs_simple_hooks/__init__.py]: I separated the conditions in commit [fc18](https://github.com/aklajnert/mkdocs-simple-hooks/pull/7/commits/fc18c56c290d0205cb56dec8b7df64e9566b1c91).

**Review by aklajnert — COMMENTED** [review]
The code looks good now, but can you rebase your fork with master once again? I think I've messed up the CI configuration and it didn't run against your PR.

**Comment by aklajnert** [comment]
Thanks @jdoiro3 for your contribution! I've just released version 0.1.4 with your change.

**Comment by jdoiro3** [comment]
This was my first open source pull request so this is exciting, even though it was a minor change. Thanks @aklajnert for making this a good experience and for starting this helpful plugin!

**Comment by aklajnert** [comment]
I hope that's a start of a great journey for you. It doesn't matter if it's a big or small change - it matter if it's useful, and in this case - it is.  
I'm happy that you had a good experience, and I wish you more of them in the future.

**Pull request merged**